website/static/js/oauthAddonNodeConfig.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * Module that controls the addon node settings. Includes Knockout view-model
```

Comment:
It's not yet used. Moving forward, it will be used for storage addons that have been moved to the new oauth system. This PR resolves a blocker for PRs for two such addons